### PR TITLE
Ravish1729/updates

### PR DESCRIPTION
**Pull request by ravish1729**
Updates
 - cid_status renamed to deal_status
 - uploadEncrypted -> response format updated
 - unnecessary console and imports removed

### CODE REVIEW

File: Lighthouse/tests/upload.test.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
-require("dotenv").config();
 const axios = require("axios");
 const { resolve } = require("path");
 const ethers = require("ethers");
-const lighthouse = require("../");
+const lighthouse = require("..");
```
Comment (nandit123) [inline Lighthouse/tests/upload.test.js:4]: seeing this first time, are you sure it's the right way to refer to dir

File: Lighthouse/tests/uploadEncrypted.test.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
-require("dotenv").config();
 const axios = require("axios");
 const { resolve } = require("path");
 const ethers = require("ethers");
-const lighthouse = require("../");
+const lighthouse = require("..");
```
Comment (nandit123) [inline Lighthouse/tests/uploadEncrypted.test.js:4]: same like before

**Review by nandit123 — COMMENTED** [review]
check the file import comment on require statement

File: Lighthouse/tests/upload.test.js (hunk 1 above)
Comment (ravish1729) [reply Lighthouse/tests/upload.test.js:4]: it works, linter changed this

**Comment by nandit123** [comment]
since comments are resolved, LGTM

**Pull request merged**